src/expr/src/interpret.rs

```diff
@@ -36,6 +36,16 @@ enum Monotonic {
     No,
 }
 
+impl From<bool> for Monotonic {
```

Comment:
The `is_monotone` functions now return `bool` instead of `Monotonic`. If we don't need the `Maybe` variant anymore, why can't we just remove the dedicated type?